unstructured/ingest/connector/confluence.py

```diff
@@ -92,37 +94,88 @@ def filename(self):
             return None
         return (
             Path(self.read_config.download_dir)
-            / self.file_meta.space_id
-            / f"{self.file_meta.document_id}.html"
+            / self.document_meta.space_id
+            / f"{self.document_meta.document_id}.html"
         ).resolve()
 
     @property
     def _output_filename(self):
         """Create output file path based on output directory, space id and document id."""
-        output_file = f"{self.file_meta.document_id}.json"
-        return Path(self.partition_config.output_dir) / self.file_meta.space_id / output_file
+        output_file = f"{self.document_meta.document_id}.json"
+        return Path(self.partition_config.output_dir) / self.document_meta.space_id / output_file
+
+    @property
+    def record_locator(self) -> t.Optional[t.Dict[str, t.Any]]:
+        return {
+            "page_id": self.document_meta.document_id,
```

Comment:
record_locator should have everything needed to programmatically get back to that source document. assuming we need more than just page_id here. some root domain to make sure we point to the right space?